pgbot/__init__.py

```diff
@@ -226,6 +226,26 @@ async def message_edit(old: discord.Message, new: discord.Message):
                 pass
 
 
+async def raw_reaction_add(payload):
+    async with db.DiscordDB("polls") as db_obj:
+        all_poll_info = db_obj.get([])
+
+        if payload.message_id in all_poll_info:
```

Comment:
This entire block below this line can be un-indented once, because it does not need to hold the DB lock nor access `db_obj`